src/Avalonia.Base/Media/FontFeatureCollection.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+using Avalonia.Collections;
+using Avalonia.Utilities;
+
+namespace Avalonia.Media;
+
+/// <summary>
+/// List of font feature settings. Syntax is the following:
+/// 
+///     Syntax 	        Value 	Start 	End 	 
+///     Setting value: 	  	  	  	 
+///     kern 	        1 	    0 	    ∞ 	    Turn feature on
+///     +kern 	        1 	    0 	    ∞ 	    Turn feature on
+///     -kern 	        0 	    0 	    ∞ 	    Turn feature off
+///     kern=0 	        0 	    0 	    ∞ 	    Turn feature off
+///     kern=1 	        1 	    0 	    ∞ 	    Turn feature on
+///     aalt=2 	        2 	    0 	    ∞ 	    Choose 2nd alternate
+///     Setting index: 	  	  	  	 
+///     kern[] 	        1 	    0 	    ∞ 	    Turn feature on
+///     kern[:] 	    1 	    0 	    ∞ 	    Turn feature on
+///     kern[5:] 	    1 	    5 	    ∞ 	    Turn feature on, partial
+///     kern[:5] 	    1 	    0 	    5 	    Turn feature on, partial
+///     kern[3:5] 	    1 	    3 	    5 	    Turn feature on, range
+///     kern[3] 	    1 	    3 	    3+1 	Turn feature on, single char
+///     Mixing it all: 	  	  	  	 
+///     aalt[3:5]=2 	2 	    3 	    5 	    Turn 2nd alternate on for range
+/// 
+/// </summary>
+public class FontFeatureCollection : AvaloniaList<string>
+{
+    public FontFeatureCollection()
+    {
+    }
+
+    public FontFeatureCollection(IEnumerable<string> features) : base(features)
+    {
+    }
+
+    /// <summary>
+    /// Parses a <see cref="FontFeatureCollection"/> string.
+    /// </summary>
+    /// <param name="s">The string.</param>
+    /// <returns>The <see cref="FontFeatureCollection"/>.</returns>
+    public static FontFeatureCollection Parse(string s)
```

Comment:
I don't think we need a special parser here.
Compiler should handle it automatically. Or, if you need an extra configuration, you can add [AvaloniaList] attribute https://github.com/AvaloniaUI/Avalonia/blob/29f4056693a57d6d7380d03611fe7c08d26113b2/src/Avalonia.Controls/DefinitionList.cs#L8